.github/copilot-instructions.md

```diff
@@ -97,6 +97,7 @@ scripts/test-nanvixd.sh 127.0.0.1:8181 bin/hello-c.elf '' '[]' 'Hello, world fro
 
 - Do not use `panic!`, `unwrap()`, or `expect()`, instead return `Result<T, E>`.
 - Avoid `unsafe` unless strictly necessary. When unavoidable, narrow its scope and document pre/post conditions.
+- Always add type annotations when defining variables.
```

Comment:
Random suggestion:
- Should we also add the need to prefix use statements with `::` like `use ::std::time...`?